source/SIL.AppBuilder.Portal.Frontend/src/ui/components/user-table/data.tsx

```diff
@@ -20,18 +22,12 @@ function mapNetworkToProps(passedProps) {
     //       is resolved
     users: [
       q => q.findRecords(USER),
-      {
-        sources: {
-          remote: {
-            settings: {
-              ...defaultSourceOptions
-            },
-            include: [MEMBERSHIPS]
```

Comment:
are the memberships not needed anymore?

Reply:
I don't see that we are using them but I guess it would be useful later when filtering

Reply:
they'll be needed if the users table is going to have any information about what orgs a user is a member of